ts-cloudfs/README.md

```diff
@@ -0,0 +1,65 @@
+# Cloud FS Sample App
+
+## Overview 
```

Comment:
❔ IMO, I dislike the double-header style and think the overview can just be under the top-level header.

Reply:
👍 will change on all of them, just aiming for consistency